src/bunit.web/Extensions/InputFile/BUnitBrowserFile.cs

```diff
@@ -0,0 +1,19 @@
+﻿#if NET5_0_OR_GREATER
+using Microsoft.AspNetCore.Components.Forms;
+
+namespace Bunit;
+
+internal class BUnitBrowserFile : IBrowserFile
+{
+    public Stream OpenReadStream(long maxAllowedSize = 512000, CancellationToken cancellationToken = default)
+    {
+        return new MemoryStream(Content);
+    }
+
+    public string Name { get; set; } = default!;
+    public DateTimeOffset LastModified { get; set; }
+    public long Size { get; set; }
+    public string ContentType { get; set; } = default!;
+    public byte[] Content { get; set; } = default!;
```

Comment:
```suggestion
    public string Name { get; set; } = default!;
    public DateTimeOffset LastModified { get; set; }
    public long Size { get; set; }
    public string ContentType { get; set; } = default!;
    public byte[] Content { get; set; } = default!;

    public Stream OpenReadStream(long maxAllowedSize = 512000, CancellationToken cancellationToken = default)
    {
        return new MemoryStream(Content);
    }
```

Small style change. In general I put properties above ctors in the code in bUnit.

A question about the properties. Why are we using `default!` instead of marking them as nullable?

Reply:
Well the `IBrowserFile` expects non-nullable types in those cases. I marked them `default!` because they are internal and we have full control over them. I will always set them.

Reply:
Looking at the code that uses this type, I think we can change this to have a ctor that takes these parameters as input, and then skip the `default!` on the properties.